### DISCUSSION TITLE
Global auto-type into terminal - Is it possible?

### DISCUSSION THREAD

**Discussion by jebbard** [keepassxreboot/keepassxc#6347] [Q&A] [answered]
Working with terminals, it is often necessary to enter passwords, e.g.

- Doing sudo
- Using ssh to connect to remote server (passphrases)

While I was still using Windows, Keepass 2.x was able to perform auto-type into terminal Windows without any issues.
Using KeepassXC (and keepass 2.x) in Fedora 33, I cannot get this to work. While global autotype in general is working for other windows, it seems like terminals are not "windows" as such. E.g. they do not appear in the window drop down when configuring auto-type for an entry. If I enter the name of the terminal window manuall, I cannot get it to work, too.

Is there a way that I can get this working, i.e. be in the terminal when the system asks for my admin password and I hit e.g. "Ctrl.+Alt+A" (my Global Autotype Hotkey) and KeepassXC autotypes the correct password into the terminal?

Would love to have that feature.

**Comment by droidmonkey** [comment, marked as answer]
You'll have to run a develop snapshot to test the new Auto-Type fixes: https://snapshot.keepassxc.org. Please report if that works for you. What terminal application are you using?

**Reply by jebbard** [reply]
Will try that and write if that helps, thanks for the link! I am using GNOME-Terminal 3.38.1 on Fedora 33. Of course, if there are other terminal applications more likely to get this working, I'd be happy to try and use those.

**Reply by droidmonkey** [reply]
You could also be running terminal as a wayland app in which case autotype will not work.

**Reply by jebbard** [reply]
I think I do not run with Wayland.

One important remark: I switched to KDE desktop due to other reasons and - wonderfully - it all works now perfectly. So for anyone experiencing the same problem: Try KDE desktop

**Reply by droidmonkey** [reply]
You most likely ran Wayland on Fedora 33 with gnome. That is the default.
